### ISSUE TITLE
Can't Delete or Modify Entry

### ISSUE THREAD

**Issue by badlandz** [GothenburgBitFactory/timewarrior#295]
I forgot to stop a task in task warrior Friday afternoon, and it span over 24 hours. It split over day in the summary, and now I can't modify or delete it.

```
$ timew delete @1
timew: /build/timew-bBlxMM/timew-1.0.0+ds.1/src/Datafile.cpp:111: void Datafile::deleteInterval(const Interval&): Assertion `_range.overlap (interval.range)' failed.
Aborted (core dumped)
```
same with modify
```
$ timew shorten @1 2hours
timew: /build/timew-bBlxMM/timew-1.0.0+ds.1/src/Datafile.cpp:111: void Datafile::deleteInterval(const Interval&): Assertion `_range.overlap (interval.range)' failed.
Aborted (core dumped)
```
I'm not quite sure what to do here
```
$ timew summary :ids 2020-02-28T00:00 - 2020-03-02T00:00:00 

Wk Date       Day ID Tags                                    Start      End     Time    Total
W9 2020-02-28 Fri @8 TASK SOMETHING A  8:58:21  9:11:34  0:13:13
                  @7 DTASK SOMETHING A  9:16:05  9:21:47  0:05:42
                  @6 TASK SOMETHING B  9:32:49  9:50:39  0:17:50
                  @5TASK SOMETHING C  10:28:09 10:33:30  0:05:21
                  @4 TASK SOMETHING D  10:50:31 11:08:27  0:17:56
                  @3 TASK SOMETHING E  11:48:32 11:53:46  0:05:14
                  @2 TASK SOMETHING F  13:07:14 14:10:20  1:03:06
                  @1TASK SOMETHING G  14:49:16  0:00:00  9:10:44 11:19:06
W9 2020-02-29 Sat @1 TASK SOMETHING G  0:00:00  0:00:00 24:00:00 24:00:00
                                                                                                                                               
                                                                                                                                       35:19:06
```

**Comment by lauft** [comment]
@badlandz as a quick workaround, you could edit the respective data file (from your output I deduce it's `~/.timewarrior/data/2020-02.data`) with a text editor and delete the respective interval by hand.

**Comment by badlandz** [comment]
@lauft Worked for me, thanks! :-)

**Comment by lauft** [comment]
@badlandz to investigate your bug it would be helpful, if you could provide the output of `timew diagnostics` or at least your Timewarrior version. Thanks

**Comment by badlandz** [comment]
```
timew 1.0.0
       Platform: Linux

Compiler:
        Version: 7.2.0
           Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
     Compliance: C++11

Build Features
          CMake: 3.9.5
     Build type: None

Configuration
  TIMEWARRIORDB: -
            Cfg: /home/xxx/.timewarrior/timewarrior.cfg (-rw- 0 bytes)
       Database: /home/xxx/.timewarrior (drwx 4096 bytes)
    Color theme: Built-in default
                 00 01 02 03 04 05 06 07 08 09 10 11 12

Extensions
       Location: /home/xxx/.timewarrior/extensions (drwx 4096 bytes)
               (None)
```
OS: Linux Mint 19.3 Tricia x86_64 
Kernel: 5.0.0-32-generic 
Shell: bash 4.4.20 

Maybe it has something to do with it rolling over the edge of a month?

**Comment by lauft** [comment]
Maybe, but your Timewarrior version is also quite outdated. I could not reproduce this with 1.2.0, the current version.

**Comment by lauft** [comment]
@badlandz As you are updating from 1.0.0, please take notice of the DB correction script which fixed a bug with exclusions present in 1.0.0: https://timewarrior.net/docs/dbcorrection.html

For safety, please backup your data before updating!
